source/server/options_impl.cc

```diff
@@ -123,13 +123,20 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv,
     throw NoServingException();
   }
 
-  if (max_obj_name_len.getValue() < 60) {
-    const std::string message = fmt::format(
-        "error: the 'max-obj-name-len' value specified ({}) is less than the minimum value of 60",
-        max_obj_name_len.getValue());
-    std::cerr << message << std::endl;
-    throw MalformedArgvException(message);
-  }
+  auto check_numeric_arg = [](bool is_error, uint64_t value, absl::string_view pattern) {
+    if (is_error) {
+      const std::string message = fmt::format(std::string(pattern), value);
+      std::cerr << message << std::endl;
+      throw MalformedArgvException(message);
```

Comment:
Can we add some options_impl_tests for this?

Reply:
There already was a test for --max-obj-name, added one for --max-stats.